mobile/mysterium/openvpn_connection_setup.go

```diff
@@ -182,16 +194,18 @@ type OpenvpnConnectionFactory struct {
 }
 
 // Create creates a new openvpn connection
-func (ocf *OpenvpnConnectionFactory) Create(stateChannel connection.StateChannel, statisticsChannel connection.StatisticsChannel) (connection.Connection, error) {
-	sessionFactory := func(options connection.ConnectOptions) (*openvpn3.Session, error) {
+func (ocf *OpenvpnConnectionFactory) Create(stateChannel connection.StateChannel, statisticsChannel connection.StatisticsChannel) (con connection.Connection, err error) {
+	sessionFactory := func(options connection.ConnectOptions) (*openvpn3.Session, *openvpn.ClientConfig, error) {
 		vpnClientConfig, err := openvpn.NewClientConfigFromSession(options.SessionConfig, "", "")
 		if err != nil {
-			return nil, err
+			return nil, nil, err
```

Comment:
You dont need explicit return variables as You have named return variables

Reply:
its inside lambda